### PR TITLE
feat: add remark-cli

### PR DESCRIPTION
**Pull request by dsully**

### CODE REVIEW

File: lua/mason-registry/remark-cli/init.lua [hunk 1]
```diff
@@ -0,0 +1,11 @@
+local Pkg = require "mason-core.package"
+local npm = require "mason-core.managers.npm"
+
+return Pkg.new {
+    name = "remark-cli",
+    desc = [[Command line interface to inspect and change markdown files with remark.]],
+    homepage = "https://github.com/remarkjs/remark/tree/main/packages/remark-cli",
+    languages = { Pkg.Lang.Markdown },
+    categories = { Pkg.Cat.LSP },
```
Comment (williamboman) [inline lua/mason-registry/remark-cli/init.lua:9]:
```suggestion
    categories = { Pkg.Cat.Formatter },
```

or am I missing something?

File: lua/mason-registry/remark-cli/init.lua [hunk 2]
```diff
@@ -0,0 +1,11 @@
+local Pkg = require "mason-core.package"
+local npm = require "mason-core.managers.npm"
+
+return Pkg.new {
+    name = "remark-cli",
+    desc = [[Command line interface to inspect and change markdown files with remark.]],
+    homepage = "https://github.com/remarkjs/remark/tree/main/packages/remark-cli",
```
Comment (williamboman) [inline lua/mason-registry/remark-cli/init.lua:7]: Websites are generally preferred when available

```suggestion
    homepage = "https://remark.js.org/",
```

File: lua/mason-registry/remark-cli/init.lua (hunk 1 above)
Comment (dsully) [reply lua/mason-registry/remark-cli/init.lua:9]: No, just copy/paste error on my part.

**Review by williamboman — APPROVED** [review]
Cool!

**Pull request merged**